src/L200DetectorConstruction.cc

```diff
@@ -0,0 +1,649 @@
+#include "L200DetectorConstruction.hh"
+#include "L200DetectorMessenger.hh"
+
+#include "G4RunManager.hh"
+#include "G4NistManager.hh"
+#include "G4Box.hh"
+#include "G4Tubs.hh"
+#include "G4Cons.hh"
+#include "G4Orb.hh"
+#include "G4Sphere.hh"
+#include "G4Trd.hh"
+#include "G4Polycone.hh"
+#include "G4AssemblyVolume.hh"
+#include "G4SubtractionSolid.hh"
+#include "G4LogicalVolume.hh"
+#include "G4VPhysicalVolume.hh"
+#include "G4PVPlacement.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4UnitsTable.hh"
+#include "G4MaterialTable.hh"
+#include "G4UnitsTable.hh"
+
+#include "G4LogicalBorderSurface.hh"
+#include "G4LogicalSkinSurface.hh"
+#include "G4OpticalSurface.hh"
+#include "G4SurfaceProperty.hh"
+#include "G4VisAttributes.hh"
+#include "G4Color.hh"
+
+#include "G4GeometryManager.hh"
+#include "G4SolidStore.hh"
+#include "G4LogicalVolumeStore.hh"
+#include "G4PhysicalVolumeStore.hh"
+#include "G4RotationMatrix.hh"
+#include "G4Region.hh"
+#include "G4ios.hh"
+
+
+#include "G4Transform3D.hh"
+
+#include <cmath>
+using namespace CLHEP;
+
+// = = = = = = = = = = = KONSTRUKTOR & DESTRUKTOR = = = = = = = = = = = = = =
+
+L200DetectorConstruction::L200DetectorConstruction()
+	: G4VUserDetectorConstruction()
+{
+	det_briefTaube = new L200DetectorMessenger(this);
+
+	worldPhys 		= NULL;
+	cryostatPhys 		= NULL;
+	lArPhys			= NULL;
+	fiberShroudInnerPhys 	= NULL;
+	fiberShroudOuterPhys	= NULL;
+	wslrCopperPhys		= NULL;
+	wslrTetraTexPhys	= NULL;
+	wslrTPBPhys		= NULL;
```

Comment:
looks like we got a running typo here. Waveshiftlengthing reflector. Also in name of physical volume (can affect macros and stuff). change or leave?

Reply:
Yes I know. If you have nothing better to do you can change it everywhere (there are also some in the messenger).